### PR TITLE
Revew part of packets

### PR DESCRIPTION
**Pull request by LFBRxD**
Revew part of packets.

### CODE REVIEW

File: src/L2dotNET/Network/GamePacketHandler.cs [hunk 1]
```diff
@@ -40,42 +40,87 @@ static GamePacketHandler()
...
+            //ClientPackets.TryAdd(0x25, typeof(RequestAnswerJoinPledge)); in L2J is RequestAnswerJoinPledge()
             ClientPackets.TryAdd(0x26, typeof(RequestWithdrawalPledge));
+            //ClientPackets.TryAdd(0x27, typeof(RequestOustPledgeMember)); in L2J is RequestOustPledgeMember()
+            //ClientPackets.TryAdd(0x28, typeof(RequestDismissPledge)); in L2J is RequestDismissPledge()
             ClientPackets.TryAdd(0x29, typeof(RequestJoinParty));
             ClientPackets.TryAdd(0x2a, typeof(RequestAnswerJoinParty));
             ClientPackets.TryAdd(0x2b, typeof(RequestWithDrawalParty));
             ClientPackets.TryAdd(0x2c, typeof(RequestOustPartyMember));
+            //ClientPackets.TryAdd(0x2D, typeof(RequestDismissParty)); in L2J is RequestDismissParty()
+            //ClientPackets.TryAdd(0x2E, typeof(DummyPacket)); in L2J is DummyPacket()
```
Comment (Elfocrash) [inline src/L2dotNET/Network/GamePacketHandler.cs]: there are also 2 more references of this in the lines below. Please keep only the valid one.

File: src/L2dotNET/Network/GamePacketHandler.cs [hunk 2]
```diff
@@ -40,42 +40,87 @@ static GamePacketHandler()
...
+            //ClientPackets.TryAdd(0x63, typeof(RequestQuestList)); in L2J is RequestQuestList
             ClientPackets.TryAdd(0x64, typeof(RequestQuestAbort));
+            //ClientPackets.TryAdd(0x65, typeof()); Packet not mapped
             ClientPackets.TryAdd(0x66, typeof(RequestPledgeInfo));
+            //ClientPackets.TryAdd(0x67, typeof(RequestPledgeExtendedInfo)); in L2J is RequestPledgeExtendedInfo
+            //ClientPackets.TryAdd(0x68, typeof(RequestPledgeCrest)); in L2J is RequestPledgeCrest
+            //ClientPackets.TryAdd(0x69, typeof(RequestSurrenderPersonally)); in L2J is RequestSurrenderPersonally
+            //ClientPackets.TryAdd(0x6a, typeof(Ride)); in L2J is Ride
+            // send when talking to trainer npc, to show list of available skills
+            //ClientPackets.TryAdd(0x6b, typeof(RequestAcquireSkillInfo)); in L2J is RequestAcquireSkillInfo
```
Comment (Elfocrash) [inline src/L2dotNET/Network/GamePacketHandler.cs]: This is also in line 125. Which one is the right one?

File: src/L2dotNET/Network/GamePacketHandler.cs [hunk 3]
```diff
@@ -40,42 +40,87 @@ static GamePacketHandler()
...
+            // send when a skill to be learned is selected
+            //ClientPackets.TryAdd(0x6c, typeof(RequestAcquireSkill)); in L2J is RequestAcquireSkill
```
Comment (Elfocrash) [inline src/L2dotNET/Network/GamePacketHandler.cs]: This also exists in the lines below. Please remove the invalid one.

File: src/L2dotNET/Network/GamePacketHandler.cs (hunk 1 above)
Comment (LFBRxD) [reply src/L2dotNET/Network/GamePacketHandler.cs]: Dummy i will review what is it.

and i put the method beside of atcual method use to compare what is the error

**Comment by Elfocrash** [comment]
Closed as the author don't fix the wrong opcodes

**Pull request closed without merging**